devtools/Tiltfile

```diff
@@ -14,16 +14,28 @@
 version_settings(constraint='>=0.22.2')
 allow_k8s_contexts('minikube')
 
+# Version configuration for components
+JOBSET_VERSION = os.getenv("JOBSET_VERSION", "v0.6.0")
```

Comment:
this needs to be `v0.8.1` in order to support ARM as well. Is there a reason to keep it lower for some specific feature set?